dev/scripts/src/util/updateEnv.ts

```diff
@@ -14,32 +14,32 @@
 import { Logger } from '@prosopo/common'
 import { at } from '@prosopo/util'
 import { getEnv } from '@prosopo/cli'
+import { glob } from 'glob'
 import dotenv from 'dotenv'
-import fg from 'fast-glob'
```

Comment:
Why changing from fast-glob to glob? I'm not against this I just don't understand why you're changing it

Reply:
Fixed